packages/trackerless-network/src/logic/inspect/InspectSession.ts

```diff
@@ -0,0 +1,49 @@
+import { EventEmitter } from 'eventemitter3'
+import { MessageRef } from '../../proto/packages/trackerless-network/protos/NetworkRpc'
+import { PeerIDKey } from '@streamr/dht'
+
+export interface Events {
+    done: () => void
+}
+
+interface InspectSessionConfig {
+    inspectedPeer: PeerIDKey
+}
+
+const createMessageKey = (messageRef: MessageRef): string => {
+    return `${messageRef.messageChainId}:${messageRef.timestamp}:${messageRef.sequenceNumber}`
```

Comment:
Need to include also publisherId as the messageChainId is unique only within one publisher. Maybe also streamPartId or needs to be included?

Reply:
Added `publisherId` to the key.

The `RandomGraphNode` and all of its components are created per stream partition so there is no need to include stream partition in the sessions